lib/pid/include/PID.h

```diff
@@ -192,7 +192,7 @@ class PID {
    * Set the output from within the PID class, needed for autotuning
    * @param output output as a double, casting done in function based on actuator type
```

Comment:
```suggestion
   * @param output output as a float, casting done in function based on actuator type
```

make sure to update comments when u make updates to code